helper.linux.fish

```diff
@@ -223,7 +223,7 @@ function oskar
   checkoutIfNeeded
   and if test "$ASAN" = "On"
     parallelism 2
-    runInContainer --privileged $UBUNTUBUILDIMAGE $SCRIPTSDIR/runTests.fish
+    runInContainer --cap-add SYS_NICE $UBUNTUBUILDIMAGE $SCRIPTSDIR/runTests.fish
   else
```

Comment:
Is this the correct place? When running ASAN, we need to trace. Unsure if this is managed by "SYS_NICE"